packages/react-instantsearch/src/connectors/connectMultiRange.js

```diff
@@ -36,46 +36,31 @@ function getCurrentRefinement(props, state) {
   return '';
 }
 
+/**
+ * MultiRange connector provides the logic to build a widget that will
+ * give the user the ability to select a range value for a numeric attribute.
+ * Ranges are defined statically.
+ * @name MultiRange
+ * @kind HOC
+ * @category connector
```

Comment:
You need because one is used to create html files, the other for the navigation on the header. Both are probably needed because at some point, we were not very mature on how to organize the doc.